src/wakefs/fuse.cpp

```diff
@@ -148,6 +149,49 @@ static bool collect_result_metadata(const std::string daemon_output, const struc
   return !result_ss.fail();
 }
 
+// Pre-resolve autofs paths through stating the paths on host machine
+static bool resolve_autofs_mounts(const std::vector<mount_op> &mount_ops) {
```

Comment:
I'm not sure how I feel about having something specific to autofs at this level -- on the one hand, it seems like requiring users to know fine details about how the filesystem is laid out would from their perspective appear to be busywork, on the other this is sitting alongside methods like `create-dir` and `tmpfs` which require a similar level of knowledge about the disk layout.  I think on the balance this is likely better than not, but I do wonder if there's any way to do this that's a bit more behind the scenes.  Then again, that's a question we've been looking at for a while, so we'd probably have come up with at least the vague shape of one by now if there was.